tools/ippfind.c

```diff
@@ -336,18 +336,14 @@ main(int  argc,				/* I - Number of command-line args */
                                argv + i)) == NULL)
             return (IPPFIND_EXIT_MEMORY);
 
-          while (i < argc)
+          do
```

Comment:
Braces around the `do` statement please...

Reply:
Fixed!